### PR TITLE
Fix(#368): Rearranging Header

### PR DESCRIPTION
**Pull request by AACHAL2303PATIL**
### *Description*
This pull request addresses issue *#368* by reorganizing the header and footer sections of our website.

### *Changes Made*
1. **Header Update**:
   - Removed the *Contacts* section from the header.
   - Replaced it with a new *Companies* section that links to the dedicated page.

2. **Footer Update**:
   - Added the *Contact* section back into the footer for easy access.

3. **Company Page**:
   - Created a dedicated page for the list of companies, sorted alphabetically.

### *Checklist*
- [x] Confirmed that the *Companies* section appears in the header.
- [x] Ensured that the *Contact* information is accessible in the footer.
- [x] Verified that the company list is alphabetized and displays correctly.

Screenshot 2024-09-27 213148
Screenshot 2024-09-27 213211

### CODE REVIEW

File: constants/navigation.ts [hunk 1]
```diff
@@ -22,8 +22,8 @@ export const navigation: NavItem[] = [
     to: '/resources',
   },
   {
-    name: 'Contact',
-    icon: 'i-ph-phone-duotone',
-    to: '/contact',
+    name: 'Companies', // Changed from 'Contact' to 'Companies'
```
Comment (MandyMeindersma) [inline constants/navigation.ts:25]: you can remove these comments! Thank you :)

**Review by MandyMeindersma — CHANGES_REQUESTED** [review]
So when I check the preview I am seeing that the comanies page isn't actually loading?

But if I hard refresh it does load:

image

**Comment by AACHAL2303PATIL** [comment]
I have made the necessary changes, and you can review them at your convenience.

**Comment by MandyMeindersma** [comment]
hmmm I am still getting that error that the comanpies page isn't work right:
image

**Comment by MandyMeindersma** [comment]
Also @AACHAL2303PATIL if you are going to do more open source contribution I would try to make your commits messages a little more descriptive rather than just the same message over and over again. It would help me review your code.

**Comment by AACHAL2303PATIL** [comment]
The page is loading correctly in my browser. could you try opening it in an incognito tab?

Screenshot 2024-10-02 111200

**Comment by arashsheyda** [comment]
@AACHAL2303PATIL the problem is if you navigate from another page to `/companies` it doesn't load.

File: pages/companies.vue [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<template>
+  <main>
+    <AppSection :group="companyGroup" />
+  </main>
+</template>
+
+<script setup lang="ts">
+import { companies } from '@/constants/companies' // Adjust the path as necessary
```
Comment (arashsheyda) [inline pages/companies.vue:8]: you don't have to include the path as it is auto-imported

**Comment by AACHAL2303PATIL** [comment]
Now check.

**Comment by arashsheyda** [comment]
also could you add `defineOgImage` for SEO, you can check the `resources` page for reference

**Comment by AACHAL2303PATIL** [comment]
Done check it.

**Comment by AACHAL2303PATIL** [comment]
I HAVE MADE THE REQUIRED CHANGES. ANYTHING ELSE YOU WANT?

**Comment by MandyMeindersma** [comment]
hey @AACHAL2303PATIL just as a heads up, when you use all caps it comes across really angry. If you are making more open source contributions I would avoid using all caps. 

We are all volunteers here so trying to keep things as positive as possible. 

In one of my first pr's I had over 75 comments on a pr. These comments really helped me be a better developer so it's something that everyone goes through :)

**Comment by MandyMeindersma** [comment]
@AACHAL2303PATIL if you click the details tab in the build info you will be able to see the error @arashsheyda and I are seeing:

image


Click details and then click "companies"

**Comment by MandyMeindersma** [comment]
I think it might just be that you should put the template at the bottom of the file? Kind of like resources. If you try to make your new company file as similar to resources I think that would really help

**Comment by arashsheyda** [comment]
@AACHAL2303PATIL thanks for your contribution! the code looks good to me

@MandyMeindersma I think the problem with the data not showing in the page might do something with the length of items (as we are using server components, I will take a look after work)

**Comment by MandyMeindersma** [comment]
Since it has been 2 weeks since there has been any changes I am going to close this pr. Let me know if you want to still work on it and I can reassign you

**Pull request closed without merging**